srcalc/src/main/java/gov/va/med/srcalc/util/ValidationCodes.java

```diff
@@ -36,6 +36,17 @@
     public static final String DUPLICATE_VALUE = "duplicateValue";
 
     /**
+     * Error code used when a rule's summand expression is not a valid Spring Expression.
+     */
+    public static final String INVALID_SUMMAND_EXPRESSION = "invalidSummandExperession";
+    
+    /**
+     * Error code used when a boolean expression of one of a rule's value matchers is not a
+     * valid Spring Expression.
+     */
+    public static final String INVALID_MATCHER_EXPRESSION = "invalidMatcherExpression";
```

Comment:
Having two different constants here is unnecessary and is inappropriate for the `util` package. One constant "invalidExpression" should be sufficient. Remind me to talk about how Spring multiplies error codes.